addOns/bruteforce/src/main/java/com/sittinglittleduck/DirBuster/HTMLparse.java

```diff
@@ -130,16 +130,15 @@ public void run() {
                                 }
 
                             } catch (MalformedURLException e) {
-                                // System.out.println("Man thats a bad url!");
+                                LOG.error("Bad URL", e);
```

Comment:
I'd suggest these be LOG.info or even LOG.debug, for `MalformedURLException` throughout.